todo_app/templates/index.html

```diff
@@ -11,7 +11,9 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+		{% for item in items %}
+			<li>{{ item.id }} - {{ item.title }} - {{ item.status }}</li>
```

Comment:
The other pages are undiscoverable right now. You have to type in the URL bar. Can you try using `<a>` elements to link to the other pages?